api/src/utils/media/dwc/dwc-archive-file.ts

```diff
@@ -85,6 +85,18 @@ export class DWCArchive {
     }
   }
 
+  // checks if this DwCArchive only has meta data
```

Comment:
nit: mind turning this into a full jsdoc comment

Reply:
done